src/assets/badgrBulkIssueTemplate.csv

```diff
@@ -1,3 +1,3 @@
-email,name,evidence
-jsmith@google.com,John,https://example.org/john-smith-project.html
-jillBrown@aol.com,Peter,
\ No newline at end of file
+E-Mail-Adresse,Vor/-Nachname
```

Comment:
It should be "Vor- / Nachname" to be grammatically correct

Reply:
I agree, but in the ticket [this](https://github.com/user-attachments/files/16351474/badgrBulkIssueTemplate_no_evidence.csv) example file was provided where the spelling is ```Vor-/Nachname```

Reply:
fixed it